packages/caliper-core/lib/manager/orchestrators/round-orchestrator.js

```diff
@@ -177,6 +177,12 @@ class RoundOrchestrator {
             logger.error(`Could not prepare worker connections: ${err.stack || err}`);
         }
 
+        process.on('SIGINT', async () => {
```

Comment:
I don't think this should be registered here. Caliper has the separation of concerns as follows
launchManager (CLI) --> caliper-engine (Core) --> round-orchestrator (Core). I think this hook should be registered in the CLI layer and then call a `stop` api on caliper-engine that in turn calls stop on the round-orchestrator. 

Caliper is an api and can be embedded in different ways the launchManager is an example of how to embed the caliper engine so I think ownership of handling signals should be done by the embedding code rather than lower down. 
Also I think it's good to expose stop methods at this level. It allows embedding code to be able to stop based on whatever reason they see fit and caliper itself then has a method point to add further code to cleanly stop on request although at the moment I think it will just be 
launchManager:
```javascript
            process.on('SIGINT', async () => {
                // support a force stop
                if (!this.terminationAlreadyRequested) {
                    this.terminationAlreadyRequested = true;
                    logger.info('request to terminate received, stopping workers');
                    try {
                        await engine.stop();
                    } catch(error) {
                        logger.error(`failed to stop workers: ${error}`);
                        process.exit(1);
                    }
                    logger.info('Workers terminated.....');
                    process.exit(0);
                } else {
                    logger.info('Termination already requested, terminating immediately');
                    process.exit(2);
                }
            });
```
caliper-engine (will need to keep a reference to round orchestrator now)
```javascript
    async stop() {
        if (this.roundOrchestrator) {
            await this.roundOrchestrator.stop();
        }
    }
```